src/elements/DsElement.js

```diff
@@ -42,8 +42,11 @@ class DsElement extends HTMLElement {
 
     $preventScroll (evt) {
         switch (evt.keyCode) {
-            case KEYS.Space:
             case KEYS.Down:
+            case KEYS.Left:
+            case KEYS.Right:
+            case KEYS.Space:
+            case KEYS.Up:
```

Comment:
This prevents scrolling if the viewport is smaller in either the X or Y dimension.